event_project/models/event_event.py

```diff
@@ -29,44 +7,67 @@
 class EventEvent(models.Model):
     _inherit = 'event.event'
 
-    project_template = fields.Many2one(
+    project_template_id = fields.Many2one(
```

Comment:
What if instead of having the project_template_id field, we had a wizard where a project template was picked and the duplication was done there?